[redis bindings][redo PR] Add GetDel support (#3552)
dapr/docs: daprdocs/content/en/reference/components-reference/supported-bindings/redis.md

You can get a record in Redis using the `get` operation. This gets a key that was previously set.

> This takes an optional parameter `delete`, which is by default `false`. When it is set to `true`, this operation will use the `GETDEL` operation of Redis. For example, it will return the `value` which was previously set and delete it.

Reviewer: ```suggestion
This takes an optional parameter `delete`, which is by default `false`. When it is set to `true`, this operation uses the `GETDEL` operation of Redis. For example, it returns the `value` which was previously set and then deletes it.
```
